tests/test_unit/test_plugin/test_collect_interface_tests.py

```diff
@@ -0,0 +1,145 @@
+import importlib
+import random
+import string
+import sys
+from textwrap import dedent
+
+import pytest
+from interface_tester.collector import collect_tests
+from interface_tester.interface_test import (
+    InvalidTestCase,
+    Role,
+    check_test_case_validator_signature,
+    get_interface_name_and_version,
+    get_registered_test_cases,
+)
+
+
+def test_signature_checker_too_many_params():
+    def _foo(a, b, c):
+        pass
+
+    with pytest.raises(InvalidTestCase):
+        check_test_case_validator_signature(_foo)
```

Comment:
Those are tests for the interface_tester project. why do we include them here instead of in the project itself